include/SFML/Audio/InputSoundFile.hpp

```diff
@@ -60,6 +59,18 @@ class SFML_AUDIO_API InputSoundFile : NonCopyable
     ////////////////////////////////////////////////////////////
     ~InputSoundFile();
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Deleted copy constructor
+    ///
+    ////////////////////////////////////////////////////////////
+    InputSoundFile(const InputSoundFile&) = delete;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Deleted copy assignment
```

Comment:
I think it is called assignment operator and not copy assignment. (Same remark for other files.)

Reply:
https://en.cppreference.com/w/cpp/language/copy_assignment

Close enough. I specifically say "*copy* assignment" because we have "*move* assignment" operators in C++11 and above.